gcn_classic_to_json/notices/GECAM_FLT/__init__.py

```diff
@@ -0,0 +1,39 @@
+from ... import utils
+
+
+def parse(bin):
+    bin[15]  # Unused. According to docs: '4 bytes for the future'
+    bin[27:39]  # Unused. According to docs: '32 bytes for the future'
+    bin[
+        11
+    ]  # Temporarily Unused. Should have a int->class_type mapping but incomplete documentation
+    bin[
+        18
+    ]  # Temporarily Unused. Should have a bit->flag mapping but incomplete documenation
+    bin[21]  # Temporarily Unused. How is it different from burst_dur?
+    bin[22:24]  # Unused. Equivalent to rate_energy_range
+    bin[
+        26
+    ]  # Temporarily Unused. Presumably which detectors activated but incomplete documenation
+    return {
+        "mission": "GECAM",
+        "id": [bin[4]],
+        "messenger": "EM",
+        "mission_type": chr(bin[19] + 64),  # mission type is A or B
+        # There is an error here; You are supposed to divide by 8640000 to get the correct value
+        # But it only seems to work if you divide by 864000000; I'm assuming it's some error in encoding the packets
+        "trigger_time": utils.datetime_to_iso8601(bin[5], bin[6] * 1e-2),
+        "trigger_type": "rate",
+        "rate_energy_range": [bin[24], bin[25]],
+        "rate_snr": bin[20] * 1e-2,
```

Comment:
This field is called `trig_signif`. Do we know that this is equivalent to SNR?

Reply:
It isn't documented, so we can't be completely certain. I was mainly going of the text notices which had the following:
TRIGGER_SIGNIF:    5.1 [sigma]  (Signif of the Trigger)
Linked [here](https://gcn.gsfc.nasa.gov/other/368.gecam)

Reply:
It's not documented, but we do have the source code.

Reply:
According to the source code: "Height of the triggering peak"

Reply:
That could mean SNR if it is the height of an SNR time series, but it could equally well be the height of the the count rate time series. @Vidushi-GitHub, @jracusin, some help here please.

Reply:
It looks a best match with rate_snr in Statistics schema. Most likely peak value from the light curve i.e. counts/s or rate.

Reply:
It doesn't sound like we reached a definite answer on this yet. @Vidushi-GitHub, how would you like to resolve this?